front/lib/api/assistant/actions/retrieval.ts

```diff
@@ -97,6 +97,10 @@ export function timeFrameFromNow(timeFrame: TimeFrame): number {
       return now - timeFrame.duration * 30 * 24 * 60 * 60 * 1000;
     case "year":
       return now - timeFrame.duration * 365 * 24 * 60 * 60 * 1000;
+    default:
+      ((x: never) => {
+        throw new Error(`Unexpected time frame unit ${x}`);
```

Comment:
nit: time frame => either TimeFrame or Timeframe depending (cf related nit on variable naming)

Reply:
looks like TimeFrame is used most of the time